src/boot/efi/boot.c

```diff
@@ -2417,6 +2633,10 @@ static void config_load_all_entries(
 
         config_load_defaults(config, root_dir);
 
+        /* find if secure boot signing keys exist and autoload them if necessary
+        otherwise creates menu entries so that the user can load them manually */
+        secure_boot_discover_keys(config, root_dir);
```

Comment:
Order in adding these to the list matters.
I'd rather these be not in the general (sorted) boot loader list. I would strongly prefer if this is called after `auto-reboot-to-firmware-setup`, which then means they are always at the very bottom of the list (though, not sorted alphabetically then).

An alternative would be to make sure these are sorted to the end in `config_entry_compare`.